doc/sly.texi

```diff
@@ -287,7 +287,7 @@ closer to Emacs' own.
 
 To understand the differences between the two projects read
 @SLY{}'s @uref{https://github.com/joaotavora/sly/blob/master/NEWS.md,,NEWS.md}
-file.  For a hand-on approach to these differences you might want to
+file.  For a hand-on approach to these differences you might want to read
```

Comment:
```suggestion
```

This is unnecessary due to how references work in `info-mode`.
`info-mode` inserts a literal "see" before any node references, see attached screenshot.
![image](https://github.com/joaotavora/sly/assets/82055622/111b3581-1c58-42bf-a4e3-ba6d36082a0e)

Reply:
If other info systems handle this differently, let me know

Reply:
This is fixed in the current version (the word "read" no longer appears).